recipes/igor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "igor" %}
+{% set version = "0.3" %}
+{% set sha256 = "b04ac38c68fb81cf3167a9981dc5a20379112d40268bb72c5a8514dc8051abba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - matplotlib
+
+test:
+  imports:
+    - igor
```

Comment:
add an import test for `igor.record`

Also, should the two scripts be tested? 

```
'bin/igorbinarywave.py',
'bin/igorpackedexperiment.py'
```

Reply:
thanks @synapticarbors for the review! I have looked into the package structure and updated the import and command tests to what seems the best to me. Feel free to comment back if you don't agree with my choices.